### PR TITLE
Fix GalleryList to use FastScrollLazyVerticalGrid when listMode

### PR DESCRIPTION
**Pull request by VDostoyevskiy**

### CODE REVIEW

File: app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryListFragment.kt [hunk 1]
```diff
@@ -836,11 +838,18 @@ fun GalleryListScreen(lub: ListUrlBuilder, navigator: NavController) {
             if (listMode == 0) {
                 val height = (3 * Settings.listThumbSize * 3).pxToDp.dp
                 val showPages = Settings.showGalleryPages
-                FastScrollLazyColumn(
+                val column = when (val detailSize = Settings.detailSize) {
```
Comment (FooIbar) [inline app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryListFragment.kt:841]: Won't recompose on setting changes

Comment (VDostoyevskiy) [reply app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryListFragment.kt]: Done

**Review by FooIbar — COMMENTED** [review]
Might do it for other settings (list thumb size, show pages etc) as well

**Comment by VDostoyevskiy** [comment]
> Might do it for other settings (list thumb size, show pages etc) as well

We need a better access pattern
Nobody wants listModeBackField together with listMode

**Comment by VDostoyevskiy** [comment]
@FooIbar PTAL
Leftover migration should land with following CL

**Review by FooIbar — APPROVED** [review]
LGTM

**Pull request merged**